source/Fleet.cpp

```diff
@@ -558,10 +562,10 @@ pair<Point, double> Fleet::ChooseCenter(const System &system)
 
 
 
-vector<shared_ptr<Ship>> Fleet::Instantiate(const Variant &variant) const
+vector<shared_ptr<Ship>> Fleet::Instantiate(vector<const Ship *> &variantShips) const
```

Comment:
Can we change the variable name variantShips here to just ships? (This function does not seem to have any variant-specific code anymore.)